.travis.yml

```diff
@@ -21,66 +21,67 @@ env:
     # Set default python version to avoid repetition later
     - BUILD_DOCS=false
     - PYTHON_VERSION=2.7
-    - MAIN_CMD="./testsuite/MDAnalysisTests/mda_nosetests --with-coverage --cover-package MDAnalysis --processes=2 --process-timeout=400 --with-memleak --with-timer --timer-top-n 50 --no-open-files"
-    - SETUP=minimal
+    - MAIN_CMD="python ./testsuite/MDAnalysisTests/mda_nosetests --processes=2 --process-timeout=400 --no-open-files --with-timer --timer-top-n 50"
+    - MAIN_CMD_OPTIONS=""
     - COVERALLS=false
     - BUILD_CMD="pip install -v package/ && pip install testsuite/"
+    - CONDA_DEPENDENCIES="numpy mmtf-python nose=1.3.7 mock six biopython networkx cython joblib nose-timer"
+    - CONDA_ALL_DEPENDENCIES="numpy mmtf-python nose=1.3.7 mock six biopython networkx cython joblib nose-timer matplotlib netcdf4 scikit-learn scipy seaborn coveralls clustalw=2.1"
+    # Install griddataformats from PIP so that scipy is only installed in the full build (#1147)
+    - PIP_DEPENDENCIES='griddataformats'
```

Comment:
Is there a reason griddataformats has to be installed on its own, and not by pip when processing the requirements?